packages/devtools/lib/src/service_manager.dart

```diff
@@ -314,11 +321,20 @@ class IsolateManager {
     _setSelectedIsolate(ref ?? isolates.first);
   }
 
-  void _setSelectedIsolate(IsolateRef ref) {
+  void _setSelectedIsolate(IsolateRef ref) async {
     if (_selectedIsolate == ref) {
       return;
     }
+
+    // Store the library uris for the selected isolate.
+    final Isolate isolate = await _service.getIsolate(ref.id);
+    selectedIsolateLibraryUris =
```

Comment:
no need to convert this list. just expose the full list of LibraryRef objects.

Reply:
that way EvalOnDartLibrary can also use it to avoid duplicated work, etc.

Reply:
done.